### ISSUE TITLE
Support for hadoop.rpc.protection=privacy in kerberized environment

### ISSUE THREAD

**Issue by alexr-bq** [vertica/spark-connector#221]
There are potential issues with running a kerberized HDFS setup with the connector and the hadoop.rpc.protection setting set to 'privacy'. This should be investigated and fixed.

**Comment by ravjotbrar** [comment]
Putting this on hold as I am unable to reproduce the issue.

**Comment by manohar3** [comment]
I got into into same issue where in connector running in executor address space and was unable to connect to secure hdfs with hadoop.rpc.protection=privacy.  This is due to fact that executor cant load have client's core-site.xml which had required setting to use privacy.  Even though i set HADOOP_CONF_DIR at driver side it seems config files are not streamed to executor hence executor were loading core-default.xml since core-site.xml was not available in classpath of exeuctor.

I was able to fix by setting --conf spark.executor.extraClassPath=/data/hadoop/conf where in /data/hadoop/conf has required core-site.xml.

Note: I was using k8s environment
